src/components/Footer/FooterLegalInfo.tsx

```diff
@@ -1,39 +1,34 @@
+import { ActiveLink } from "@/components/shared/ActiveLink";
 import { Routes } from "@/routes";
-import Link from "next/link";
 
-export const FooterLegalInfo = () => {
-  return (
-    <div className="mt-8 flex w-full flex-col gap-5 text-center">
-      <p className="flex justify-center gap-5">
-        <Link
-          href={Routes.POLICY}
-          className="text-16 transition-opacity hover:opacity-80"
-          aria-label="Przejdź do strony Polityka prywatności"
-          title="Przejdź do strony Polityka prywatności"
-        >
-          Polityka prywatności
-        </Link>
+export const FooterLegalInfo = () => (
```

Comment:
I don't think that Active Links are needed here. But we can leave it as it is. It may be useful for some people.